### PR TITLE
[ADF-5146] Upgrade to Angular 9

### PR DESCRIPTION
**Pull request by DenysVuika**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

- upgrade to Angular 9
- cleanup of package.json scripts (remove unused ones, simplify existing)
- improved demo shell configs (tsconfig, tslint, etc)
- fix karma configuration for "insights" library
- all packages now work with `ng build <name>` commands
- all packages now work with `ng test <name>` commands, with proper watch modes to restart tests

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://issues.alfresco.com/jira/browse/ADF-5146

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -54,10 +54,14 @@ jobs:
     include:
         # Run Only for any PR
         - stage: Lint & Build Dist
-          name:  Lint & Build Dist
+          name:  Build
           script:
-            - ./scripts/travis/build/build.sh || exit 1;
-            - ./scripts/lint.sh || exit 1;
+            - ./scripts/travis/build/build.sh
+        - stage: Lint & Build Dist
```
Comment (mauriziovitale) [inline .travis.yml:60]: why did you split it? in this way, we will wait more to spawn the VM compared to the time the lint takes. This is the reason why we merged the build and lint

Comment (DenysVuika) [reply .travis.yml:60]: with ngcc, the whole run takes 30 minutes, I've split it to be 10 mins approx

File: .travis.yml [hunk 2]
```diff
@@ -58,22 +57,23 @@ jobs:
...
             - ./scripts/lint.sh
 
 
         # Run Only on develop and master
-        - stage: Lint & Build Dist & Release
-          name:  Lint & Build Dist & Release
+        - stage:
+
+          name: Npm Release
+          before_script: ./scripts/ci/utils/artifact-from-s3.sh -a "$S3_DBP_FOLDER/alfresco-libs.tar.bz2" -o "./lib/dist"
```
Comment (mauriziovitale) [inline .travis.yml:73]: @eromano can you check this change. 
We use the same Lint & Build Dist to build and link the artifact will store in S3 and the Release job will fetch the artifact from S3 and publish it on npm

**Comment by DenysVuika** [comment]
Closing as we have ng 10 pr

**Pull request closed without merging**